src/main/kotlin/com/github/hanseter/json/editor/ui/CustomTreeTableView.kt

```diff
@@ -0,0 +1,115 @@
+package com.github.hanseter.json.editor.ui
+
+import javafx.scene.control.TreeItem
+import javafx.scene.control.TreeTableView
+import javafx.scene.input.KeyCode
+import javafx.scene.input.KeyEvent
+
+class CustomTreeTableView<S> : TreeTableView<S>() {
```

Comment:
Yes this is a `CustomTreeTableView` but that doesn't tell me anything about what it actually does.
What it was customized for.
A more expressive name might be `NavigatableTreeTableView`.

But I would argue an even better solution would be to not subclass at all.
This could very be a method that simply sets the listeners as they are set in the constructor.
The method could be called `addNavigationToTreeTable` or something.